roles/setup_hammerdb/tasks/update_pg_hba.yml

```diff
@@ -0,0 +1,35 @@
+---
+
+- name: Reference pg_type variables
+  include_vars: "{{ role_path }}/../init_dbserver/vars/{{ pg_type }}.yml"
+
+- name: Initialize the hba_entries variable
+  set_fact:
+    hba_entries: >-
+      [
+        {
+          'users': 'all',
+          'databases': 'all',
+          'contype': 'host',
+          'source': '{{ hostvars[inventory_hostname].hammerdb_server_private_ip | default("127.0.0.1") }}/32',
+          'method': 'trust'
```

Comment:
Looks a bit permissive.

Reply:
Yeah, I admit I made that decision for ease of development.  Do we need to use passwords and SSL connections?

Reply:
I think we can live with this for the moment.